People/README.md

```diff
@@ -78,10 +78,17 @@ This sample uses the Gradle build system. To build this project, use the
     class BubbleActivity : AppCompatActivity() {
       override fun onCreate(...) {
       ...
-      val displayId = this.display.displayId
-      val isBubbled = displayId != Display.DEFAULT_DISPLAY
-      ...
-     } 
+        if (Build.VERSION.SDK_INT == 29) {
```

Comment:
I get what you mean, but `displayID` and `isBubbled` here are not accessible from outside the if scope. I suppose something like this is closer to actual code.

```
val isBubbled = if (Build.VERSION.SDK_INT >= 31) {
  isLaunchedFromBubble()
} else {
  val displayId = if (Build.VERSION.SDK_INT >= 30) {
    display.displayId
  } else {
    windowManager.defaultDisplay.displayId
  }
  displayId != Display.DEFAULT_DISPLAY
}
```

Reply:
right..I wasn't thinking too much about usability but primarily showing the means of the getting the value determining if the activity is in a bubble. But this is much better. Thanks